crypto/threads_win.c

```diff
@@ -133,4 +133,10 @@ int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock)
     return 1;
 }
 
+int openssl_init_fork_handlers(void)
+{
+    /* Is failing the right thing to do? */
+    return 0;
```

Comment:
Hmmm, I don't think it should fail in this case...  or in that case, it should fail for VMS in the `threads_pthread.c` as well.